src/components/EmojiPicker/EmojiPickerButton.tsx

```diff
@@ -24,12 +24,12 @@ type EmojiPickerButtonProps = {
     /** Emoji popup anchor offset shift vertical */
     shiftVertical?: number;
 
-    onModalHide: EmojiPickerAction.OnModalHideValue;
+    onModalHide?: EmojiPickerAction.OnModalHideValue;
 
     onEmojiSelected: EmojiPickerAction.OnEmojiSelected;
 };
 
-function EmojiPickerButton({isDisabled = false, id = '', emojiPickerID = '', shiftVertical = 0, onModalHide, onEmojiSelected}: EmojiPickerButtonProps) {
+function EmojiPickerButton({isDisabled = false, id = '', emojiPickerID = '', shiftVertical = 0, onModalHide = () => {}, onEmojiSelected}: EmojiPickerButtonProps) {
```

Comment:
After selecting an emoji, we won't focus back the composer. TBH, I'm feeling a bit disquiet about changing this. 

https://github.com/user-attachments/assets/c1e57a76-893f-44d2-bb09-984c2c45d411

Reply:
It is also a part of the scope of fixing. From my point of view, it is not a problem and it is good as well. But let's decide https://github.com/Expensify/App/pull/48415#discussion_r1751702038 first.